src/main/res/values/strings.xml

```diff
@@ -139,23 +139,23 @@
   <string name="unknown_error_description">Unknown error.</string>
   <string name="unknown_format">Unknown: %s</string>
   <string name="unsigned">Unsigned</string>
-  <string name="unsigned_description">Unsigned. The list of applications could not be verified. Be careful with
-    applications downloaded from unsigned repositories.</string>
+  <string name="unsigned_description">Unsigned. Could not verify the application list. Be careful downloading
+    applications from unsigned repositories.</string>
   <string name="unstable_updates">Unstable updates</string>
-  <string name="unstable_updates_summary">Suggest updates to unstable versions</string>
+  <string name="unstable_updates_summary">Suggest installing unstable versions</string>
   <string name="unverified">Unverified</string>
   <string name="update">Update</string>
-  <string name="update_notifications">Update notifications</string>
-  <string name="update_notifications_summary">Show a notification when updates are available</string>
+  <string name="update_notifications">Notifications about newer applications</string>
+  <string name="update_notifications_summary">Show a notification when new versions are available</string>
   <string name="updates">Updates</string>
   <string name="updates_empty_summary">All applications up to date</string>
   <string name="username">Username</string>
-  <string name="username_is_not_specified">Username is not specified</string>
-  <string name="validation_index_error_description">Failed to validate the index.</string>
-  <string name="validation_package_error_description_format">Failed to validate the package: %s.</string>
-  <string name="validation_error_hash_lower">invalid hash</string>
-  <string name="validation_error_integrity_lower">integrity check failed</string>
-  <string name="validation_error_permissions_lower">missing permissions</string>
+  <string name="username_is_not_specified">Username missing</string>
+  <string name="validation_index_error_description">Could not validate the index.</string>
+  <string name="validation_package_error_description_format">Could not validate the package: %s.</string>
```

Comment:
```suggestion
  <string name="validation_package_error_description_format">Application could not be validated: %s.</string>
```

Reply:
"Package" is a specific _version of application bundled into signed ZIP archive_. So "application" isn't good here, "package" should be used instead.